webapp/src/main/resources/application.sql.yml

```diff
@@ -67,3 +68,13 @@ sql:
 
     sqlFindByCreator: >-
       SELECT * FROM upload_student_group_batch WHERE creator=:creator
+
+  import:
+    createQuery: >-
+      INSERT INTO import (status, content, contentType, digest, batch) VALUES
+        (0, 5, 'admin group modification', :digest, :batch)
```

Comment:
I'm not thrilled with the ongoing abuse of the `contentType` column. I don't really have a better suggestion just bringing it up as a discussion topic.